test/functional/Valhalla/src/org/openj9/test/lworld/NullRestrictedTypeOptTests.java

```diff
@@ -0,0 +1,341 @@
+/*******************************************************************************
+ * Copyright IBM Corp. and others 2022
```

Comment:
Sorry I was just updating other test files and noticed the copyright year. Should this be `and others 2023`?

Reply:
Good point!

Reply:
LGTM